creator/ingest_runs/data_generator/study_generator.py

```diff
@@ -0,0 +1,504 @@
+"""
```

Comment:
I took all of the individual methods you had and put them in one class so its easier to see what's going on and call in an entry point function that will do the data generation and the ingestion of the files into Data Service.